todo_app/trello_api.py

```diff
@@ -0,0 +1,110 @@
+import os
+import requests
+import sys
+
+base_query = { 'key': os.environ.get('TRELLO_API_KEY'), 'token': os.environ.get('TRELLO_API_TOKEN') }
+board_name = 'Task Manager'
+todo_list_name = 'To Do'
+done_list_name = 'Done'
+
+# Set up initial board with a to do list and a done list.
+def init_boards():
+    board_id = create_board(board_name)
+    create_list(board_id, todo_list_name)
+    create_list(board_id, done_list_name)
+
+# Create board if it doesn't exist.
+def create_board(board_name):
+    board_id = find_board(get_boards(), board_name)
+    if board_id != '':
+        return board_id
+    url = 'https://api.trello.com/1/boards/'
+    query = dict(base_query)
+    query['name'] = board_name
+    response = requests.post(url, params=query)
+    result = response.json()
+    return result['id']
+
+def get_boards():
+    username = os.environ.get('TRELLO_USERNAME')
+    url = f'https://api.trello.com/1/members/{username}/boards'
```

Comment:
You can use `https://api.trello.com/1/members/me/boards`, that way you don't need to know the username